src/compute/app_runner.rs

```diff
@@ -7,6 +7,19 @@ use crate::compute::{
 };
 use log::{error, info};
 
+/// Represents the different exit modes for a process or application.
+///
+/// Each variant is explicitly assigned an `i32` value, and the enum
+/// uses `#[repr(i32)]` to ensure its memory representation matches C-style enums.
+#[cfg_attr(test, derive(Debug, Clone, PartialEq))]
```

Comment:
`Clone` seems unnecessary here. If it was, do not forget the advice given to us to order them in lexicograpic order for standard ones.
```suggestion
#[cfg_attr(test, derive(Debug, PartialEq))]
```